admin/pre_reg/views.py

```diff
@@ -49,7 +49,9 @@ class DraftListView(PermissionRequiredMixin, ListView):
     def get_queryset(self):
         return DraftRegistration.objects.filter(
             registration_schema=get_prereg_schema(),
-            approval__isnull=False
+        ).exclude(
+            approval__isnull=True,
```

Comment:
Isn't exclude(approval__isnull=True) the same as filter(approval__isnull=False)? 
I think just adding the exclude(registered_node__isnull=True) is sufficient.

Reply:
I think I realize what you are trying to do, but it requires changing your query syntax to something like:
```
prereg_query = Q(registration_schema=get_prereg_schema())
approved_query = Q(approval__isnull=False)
rejected_query = Q(approval__isnull=True, registered_node__isnull=False)
return DraftRegistration.objects.filter(prereg_query & (approved_query | rejected_query))

```